daisy/workflow.go

```diff
@@ -312,6 +312,9 @@ func (w *Workflow) GetStepTimeRecords() []TimeRecord {
 }
 
 func (w *Workflow) cleanup() {
+	// cleanup cache so the next workflow can have a clean env
+	defer cleanupCache()
```

Comment:
Why is this needed? What does "next workflow" mean? I thought caches are workflow specific, not global.

Reply:
It's global in current implementation. One reason is that a struct "disk" doesn't rely on a workflow to behave.